docs/interpreter/kylin.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: "Apache Kylin Interpreter for Apache Zeppelin"
+description: "Apache Kylin™ is an open source Distributed Analytics Engine designed to provide SQL interface and multi-dimensional analysis (OLAP) on Hadoop supporting extremely large datasets, original contributed from eBay Inc.
+."
+group: interpreter
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+# Apache Kylin Interpreter for Apache Zeppelin
+
+<div id="toc"></div>
+
+## Overview
+[Apache Kylin](https://kylin.apache.org/) is an open source Distributed Analytics Engine designed to provide SQL interface and multi-dimensional analysis (OLAP) on Hadoop supporting extremely large datasets, original contributed from eBay Inc. The interpreter assumes that Apache Kylin has been installed and you can connect to Apache Kylin from the machine Apache Zeppelin is installed.  
+To get start with Apache Kylin, please see [Apache Kylin Quickstart](https://kylin.apache.org/docs15/index.html).
```

Comment:
@AhyoungRyu  Thank you for your review, for 'doc15' you mentioned, now 'doc15' is a stable link for kylin, we do not have other link like 'latest' now, if there's update on this in future, I will also update on Zeppelin side.

And for 'Default Value', if you check again you can see first column represents property name, and sceond column represents default value，is that what you mean? thanks!